backend/tests/test_endpoints_moodle.py

```diff
@@ -216,11 +203,311 @@ def test_create_course_build_duplicate_shortname_moodle(
     assert data["course_id"] is None
     assert data["course_enrollment_url"] is None
     assert data["course_enrollment_key"] is None
-    assert data["school_district"] == "snowfall_isd"
+    assert data["school_district_name"] == "snowfall_isd"
     assert data["academic_year"] == "AY 2024"
     assert data["academic_year_short"] == "AY24"
     assert data["status"] == "created"
-    assert data["creator"] == "manager@rice.edu"
+    assert data["creator_email"] == "manager@rice.edu"
+
+
+def test_get_course_build_by_academic_year_and_instructor_email(
```

Comment:
I don't think these test cases are really giving the confidence that we'd like as far as the query parameters filtering as expected. For example, if I comment out [these lines](https://github.com/openstax/rope/blob/237963955448f9b057d6dc2228c37eb7c4443706/backend/api/database.py#L181-L187), all tests still pass. I would suggest setting up a shared fixture that has multiple course builds with varying values for district / AY. Then:

* `test_get_all_course_builds` should return all
* `test_get_course_build_by_instructor_email` should return some subset 
* `test_get_course_build_by_academic_year` should return on some subset
* `test_get_course_build_by_academic_year_and_instructor_email` should return on the intersection of the two previous but a smaller set

A good litmus test is if you comment lines in `database.get_course_build` which add filters, some test cases should fail.

Reply:
I created a fixture that creates multiple course builds and passed that fixture to the four tests. Using the litmus test, I confirmed that we do get expected failures when the filters are commented out. One note - the comment above mentions using varying districts / AYs. I think you meant instructor emails / AYs (which I did), since those are the two filters we are testing. If you want varying districts as well, I can do that.

Reply:
No, I mis-wrote. AY and emails sounds right.